joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/frontend/XTypeHintCallLinker.scala

```diff
@@ -31,11 +31,11 @@ abstract class XTypeHintCallLinker(cpg: Cpg) extends CpgPass(cpg) {
   def calleeNames(c: Call): Seq[String] =
     (c.dynamicTypeHintFullName :+ c.typeFullName).filterNot(_.equals("ANY")).distinct
 
-  private def callees(names: Seq[String]): List[Method] = cpg.method.fullNameExact(names: _*).toList
+  def callees(names: Seq[String]): List[Method] = cpg.method.fullNameExact(names: _*).toList
```

Comment:
Seems that this is not overridden anymore
```suggestion
  private def callees(names: Seq[String]): List[Method] = cpg.method.fullNameExact(names: _*).toList
```

Reply:
Changes in this file are no longer needed after [this](https://github.com/joernio/joern/blob/437c4a73d46ed25bfe549861481ac9c6bf05e929/joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitor.scala#L1690) change. Restored this code to what it is in `master`